wonjong/controllers/likeController.js

```diff
@@ -0,0 +1,21 @@
+const likeService = require('../services/likeService');
+
+const createLike = async (req, res) => {
+  try {
+    const { userId, postId } = req.params;
+    if (!userId || !postId) {
+      return res.status(400).json({ message: 'KEY_ERROR' });
+    }
+    await likeService.createLike(userId, postId);
+    return res.status(201).json({
```

Comment:
가독성을 위해 구문들 사이에 공백을 추가해 주면 좋을 것 같습니다.
```suggestion
    const { userId, postId } = req.params;
    
    if (!userId || !postId) {
      return res.status(400).json({ message: 'KEY_ERROR' });
    }
    
    await likeService.createLike(userId, postId);
    return res.status(201).json({
```